package.json

```
@@ -8,8 +8,12 @@
   },
   "scripts": {
     "build": "webpack",
-    "lint": "eslint --ext js,jsx src"
+    "lint": "eslint --ext js,jsx src/web --config ../graylog2-server/graylog2-web-interface/.eslintrc.json",
+    "lint:path": "eslint --ext js,jsx --config ../graylog2-server/graylog2-web-interface/.eslintrc.json",
+    "test": "jest",
+    "flow-check": "flow check"
   },
+
```

Try out setting the `eslintConfig` and see if this still works as expected

```suggestion
    "lint": "eslint --ext js,jsx src/web",
    "lint:path": "eslint --ext js,jsx",
    "test": "jest",
    "flow-check": "flow check"
  },
  "eslintConfig": {
    "extends": [
      "../graylog2-server/graylog2-web-interface/.eslintrc.json"
    ]
  },
```